rpcq/_base.py

```diff
@@ -163,7 +164,10 @@ def from_msgpack(b):
     # Docs for raw parameter are somewhat hard to find so they're copied here:
     #   If true, unpack msgpack raw to Python bytes (default).
     #   Otherwise, unpack to Python str (or unicode on Python 2) by decoding with UTF-8 encoding (recommended).
-    return msgpack.loads(b, object_hook=_object_hook, raw=False)
+    #   In msgpack >= 0.6, max_xxx_len is reduced from 2 GB to 1 MB, so we set the relevant ones
```

Comment:
1 GB?

Reply:
no it was 2, I just decided I like 1 GB better

Reply:
No, I mean, what's 1 MB doing here?

Reply:
oh -- because they reduced the default from 2 GB to 1 MB in version 0.6

Reply:
that's because it is! anyone have a preferred size? because 2 GB is pretty darn slow to serialize. but if that's the consensus I'll put it back to 2!

Reply:
done